src/internal_scc.rs

```diff
@@ -1 +1,134 @@
+pub struct Csr<E> {
+    start: Vec<usize>,
+    elist: Vec<E>,
+}
 
+impl<E> Csr<E>
+where
+    E: Copy + Clone,
```

Comment:
You can require only `Copy`, because `trait Copy: Clone`.  
```suggestion
    E: Copy,
```